foundationsync/indico_foundationsync.py

```diff
@@ -104,9 +112,9 @@ def _parse_room_data(self, raw_data, coordinates, room_id):
         data['capacity'] = int(raw_data['CAPACITY']) if raw_data['CAPACITY'] else None
         data['surface_area'] = int(raw_data['SURFACE']) if raw_data['SURFACE'] else None
         data['division'] = raw_data.get('DEPARTMENT')
-        data['telephone'] = raw_data.get('TELEPHONE')
-        data['key_location'] = raw_data.get('WHERE_IS_KEY')
-        data['comments'] = raw_data.get('COMMENTS')
+        data['telephone'] = raw_data.get('TELEPHONE') if raw_data.get('TELEPHONE') else ''
```

Comment:
This a normal `dict` right? Then you could do `raw_data.get('TELEPHONE', '')`

Reply:
might be NULL in the DB - in that case `raw_data.get('TELEPHONE') or ''` should work though

Reply:
Oh right. How is the DB NULL represented? Is it falsy?

Reply:
`None` i guess